### PR TITLE
[Compiler Plugin] - CU#8677bhj55 - Dependent suspend continuations with two arguments

### PR DESCRIPTION
**Pull request by pabloconde**
This PR adds some missing test cases for dependent continuations with two arguments.

### CODE REVIEW

File: continuationsPluginExample/src/main/scala/examples/Main.scala [hunk 1]
```diff
@@ -6,13 +6,8 @@ import continuations.jvm.internal.ContinuationImpl
...
 //  println(result)
-  val result2 = programSuspendContinuationNoParamResume
-  println(result2)
-  val result3 = programSuspendContinuationNoParamNoSuspendContinuation
-  println(result3)
-  val result4 = programSuspendContinuationNoParamResumeIgnoreResult
-  println(result4)
-  println(programSuspendContinuationParamDependent)
+
+  println(programSuspendContinuationTwoParamsDependent)
```
Comment (gregorath) [inline continuationsPluginExample/src/main/scala/examples/Main.scala:10]: should we not keep the existing ones too?

Comment (jackcviers) [reply continuationsPluginExample/src/main/scala/examples/Main.scala:10]: I think we should.

Comment (jackcviers) [reply continuationsPluginExample/src/main/scala/examples/Main.scala:10]: Also: there should be a standalone main for `programSuspendContinuationTwoParamsDependent` like we have for the others we just demoed.

**Review by jackcviers — CHANGES_REQUESTED** [review]

**Review by gregorath — APPROVED** [review]
👍

**Review by jackcviers — APPROVED** [review]

**Pull request merged**